app/views/trainees/diversity/disability_details/edit.html.erb

```diff
@@ -11,15 +11,23 @@
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <%= form_with(model: @disability_detail, url: trainee_diversity_disability_detail_path(@trainee), method: :put, local: true) do |f| %>
       <%= f.govuk_error_summary %>
-
-      <%= f.govuk_collection_check_boxes :disability_ids,
-          @disabilities,
-          :id,
-          :name,
-          :description,
+      
+      <%= f.govuk_check_boxes_fieldset :disability_ids, 
           legend: { text: t("components.page_titles.trainees.diversity.disabilities.edit"), tag: "h1", size: "l" },
           hint: { text: "Select all that apply" },
-          classes: "disability" %>
+          classes: "disability" do %>
+        
+        <% @disabilities.each_with_index do |disability, index|  %>
+          <% if disability.name != Diversities::OTHER  %>
+            <%= f.govuk_check_box :disability_ids, disability.id, label: { text: disability.name, size: "s" }, hint: { text: disability.description }, link_errors: index.zero? %>
+          <% else %>
```

Comment:
Does 'other' always need to be the last option in the list? This could render it anywhere, depending on the order of `@disabilities`. It's not really a big deal for now since 'other' happens to be the last alphabetically!

Reply:
Yes, based on the UI it's the last option in the list. So this should render any other option before it